.github/build.yaml.gomplate

```diff
@@ -507,10 +507,13 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Export cos version
+        run: |
+          echo "COS_VERSION=$(yq r packages/cos/collection.yaml 'packages.[0].version')" >> $GITHUB_ENV
```

Comment:
It won't work there is no `yq` in this environment :sweat: We overlapped, I did the same an equivalent commit 5mins ago [here](https://github.com/rancher-sandbox/cOS-toolkit/commit/052cf819f4ede0ef954bd84c95513fdf9eabddc2)

Reply:
closing this one then!